lib/nandi/multi_db_generator.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Nandi
+  module MultiDbGenerator
+    def self.included(base)
+      base.class_option :database, type: :string, desc: "Database to migrate in multi-database mode. " \
+                                                        "If not specified, uses specified default or primary database"
+    end
```

Comment:
We could also do it this way

```suggestion
     module MultiDbGenerator
    extend ActiveSupport::Concern

    included do
      class_option :database, type: :string, desc: "..."
    end
  end
```